common/persistence/persistenceInterface.go

```diff
@@ -70,6 +70,10 @@ type (
 		//  - number of rows actually deleted, if limit is honored
 		//  - UnknownNumRowsDeleted, when all rows below value are deleted
 		CompleteTasksLessThan(ctx context.Context, request *CompleteTasksLessThanRequest) (int, error)
+		// GetOrphanTasks returns tasks that exist as records in the database but are part of task lists which
+		// _do not_ exist in the database. They are therefore unreachable and no longer represent valid items
+		// that can be legitimately acted upon.
+		GetOrphanTasks(ctx context.Context, request *GetOrphanTasksRequest) (*GetOrphanTasksResponse, error)
```

Comment:
Would you mind adding a persistence test for this API? It should be pretty straightforward to write. We have a convention of high test coverage for persistence as it's very critical layer for Cadence. The test will protect us from breaking it especially when people do refactoring.

https://github.com/uber/cadence/blob/master/common/persistence/persistence-tests/matchingPersistenceTest.go
You can see examples for other APIs.